### PR TITLE
Added details of absence counts to MyCalendar page, including year control

### PR DESCRIPTION
**Pull request by AlexJYoungman**
connects to #13

### CODE REVIEW

File: app/styles/css/calendar.css [hunk 1]
```diff
@@ -4,12 +4,13 @@ body {
 }
 
 .calendar-details {
-    width: 100%;
-    display: block;
     color: #666666;
```
Comment (TheGarrett) [inline app/styles/css/calendar.css]: file is indented by 4 instead of 2 spaces.

File: app/components/MyCalendar.js [hunk 2]
```diff
@@ -1,20 +1,279 @@
...
+}
 
 var MyCalendar = React.createClass({
+  // Method to count the number of days  for each absence
+  getDays: function () {
+    // Currently hardcoded data, eventaully will be retrieved from the database
+    jsonData = {
+      'name': 'John Smith',
+      'absences': [
+        {
```
Comment (TheGarrett) [inline app/components/MyCalendar.js]: This is just me being picky but could store json data in its own json page? then call that page.

File: app/components/MyCalendar.js [hunk 3]
```diff
@@ -1,20 +1,279 @@
...
+          </span>
+          <span className='add-absence'>
+            <button className="add-absence-button">CREATE ABSENCE<span className="glyphicon glyphicon-plus"></span>
+          </button>
+          <div>or click on the calendar below</div>
+        </span>
+      </span>
+      <YearCalendar year={this.state.year}/>
+    </div>
+  );
```
Comment (TheGarrett) [inline app/components/MyCalendar.js]: your indentation doesn't line up here.

File: app/components/MyCalendar.js (hunk 2 above)
Comment (AlexJYoungman) [inline app/components/MyCalendar.js]: Potentially but this is only temporary until we have a db.

**Pull request merged**